### PR TITLE
feat/add oneclick exception

### PR DESCRIPTION
**Pull request by mvarlic**
This PR adds new exception classes.

### CODE REVIEW

**Review by victormendoza96 — APPROVED** [review]

File: shared/Exceptions/Oneclick/CreateTransactionOneclickException.php
Comment (Matiasnickolas) [inline shared/Exceptions/Oneclick/CreateTransactionOneclickException.php:1]: Oneclick handle a create transaction exception?

Comment (isabellaahc) [inline shared/Exceptions/Oneclick/CreateTransactionOneclickException.php:1]: Oneclick flow not include a create transaction

File: shared/Exceptions/Oneclick/StartOneclickException.php
Comment (isabellaahc) [inline shared/Exceptions/Oneclick/StartOneclickException.php:1]: What's the difference between this class and StartInscriptionOneclickException?

**Review by isabellaahc — COMMENTED** [review]
Please check the comment in: 
- shared/Exceptions/Oneclick/CreateTransactionOneclickException.php
- shared/Exceptions/Oneclick/StartOneclickException.php

**Pull request closed without merging**